source/connections-survive-step-down/tests/README.rst

```diff
@@ -159,3 +165,14 @@ Do we need to wait for re-election after the first test?
 ``````````````````````````````````````````````````````````
 
 Since test setup requires creation of a collection, a primary must exist, so subsequent tests will block in server selection until a primary is available again.
+
+
+Why do tests check for a successful insert operation in addition to checking that the pool was updated appropriately?
+```````````````````````````````````````````````````````````````````````````````````````````````````````````````````````
```

Comment:
I think two backticks need to be trimmed here. GitHub may render it, but it's technically an RST warning if the text and header underline don't match up.

Reply:
Done. Also did the same for the existing Q&A item (it too had 2 extra backticks).